package.json

```diff
@@ -1,34 +1,9 @@
 {
   "name": "cuttle",
-  "version": "3.0.2",
-  "private": true,
   "description": "The deepest card game under the sea",
-  "scripts": {
-    "build": "vue-cli-service build && cp ./client/public/favicon.ico ./assets",
-    "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint --no-fix",
-    "debug": "node debug app.js",
-    "e2e:client": "npx cypress run --config-file tests/cypress.config.js",
-    "e2e:gui": "ENABLE_VUE_DEVTOOLS=true npx cypress open --config-file tests/cypress.config.js",
-    "e2e:mockprod": "CUTTLE_ENV=production npm run e2e:server",
-    "e2e:server": "VUE_APP_API_URL=http://localhost:1337 npx cypress run --config-file tests/cypress.config.js",
-    "format:js": "./scripts/format-project-js",
-    "lint:fix": "vue-cli-service lint",
-    "start": "node app.js",
-    "start:client": "concurrently \"ENABLE_VUE_DEVTOOLS=true vue-cli-service serve\" \"npm run start:devtools\"",
-    "start:dev": "concurrently \"npm run start:server\" \"ENABLE_VUE_DEVTOOLS=true npm run start:client\" \"npm run start:devtools\"",
-    "start:devtools": "vue-devtools",
-    "start:mockprod": "npm run build && CUTTLE_ENV=production npm run start:server",
-    "start:server": "node app.js"
-  },
-  "main": "app.js",
+  "version": "3.0.2",
```

Comment:
I know we recently did some dev upgrades, might be worth bumping this to a new version soon. I know it's not a big "app" feature, but there's an argument to be made for a 3.1.X release.